source/Nuke.Common.Tests/SettingsTest.cs

```diff
@@ -42,6 +42,10 @@ public void TestCommon()
             settings.ProcessEnvironmentVariables.Should().ContainSingle(x => x.Key == "key" && x.Value == "value");
             settings.ProcessExecutionTimeout.Should().Be(1_000);
             settings.ProcessArgumentConfigurator.Invoke(new Arguments()).RenderForOutput().Should().Be("/switch");
+
+            settings = settings.SetProcessExecutionTimeout(TimeSpan.FromMilliseconds(500));
```

Comment:
can you please put this into the already existing _act_ part?

Reply:
just remove the old `Set...Timeout` via milliseconds.

Reply:
done